app/models/tortoise/user.py

```diff
@@ -2,94 +2,33 @@
 This module is made to provide different user models to the API.
 It contains pydantic and Tortoise models.
 """
-from typing import Annotated, Optional
 
-import bcrypt
-from fastapi import Depends
-from fastapi.security import OAuth2PasswordBearer
-from passlib.context import CryptContext
-from tortoise import fields
+from tortoise.fields import Field, IntField, ForeignKeyRelation, CharField, TextField, ForeignKeyField
 from tortoise.models import Model
 
-from app.services.Tokens import AvailableTokenAttributes, Token
-from app.models.pydantic.TokenModel import PydanticToken
-from app.utils.type_hint import JWTData
+from app.models.tortoise.role import RoleInDB
 
-pwd_context:    CryptContext         = CryptContext(schemes=["bcrypt"], deprecated="auto")
-oauth2_scheme:  OAuth2PasswordBearer = OAuth2PasswordBearer(tokenUrl="/auth/login")
 
 
 class UserInDB(Model):
     """
     This class is designed to design the table inside of the database.
     It contains all the columns necessary to store the user.
     """
-    id = fields.IntField(primary_key=True)
-    login = fields.CharField(unique=True, required=True, max_length=128)
-    firstname = fields.CharField(max_length=128)
-    lastname = fields.CharField(max_length=128)
-    mail = fields.CharField(unique=True, required=True, max_length=128)
-    hash = fields.TextField(required=True)
+    id        : Field[int]                   = IntField(primary_key=True)
+    login     : Field[str]                   = CharField(unique=True, required=True, max_length=128)
+    firstname : Field[str]                   = CharField(max_length=128)
+    lastname  : Field[str]                   = CharField(max_length=128)
+    mail      : Field[str]                   = CharField(unique=True, required=True, max_length=128)
+    hash      : Field[str]                   = TextField(required=True)
+    role      : ForeignKeyRelation[RoleInDB] = ForeignKeyField("models.RoleInDB", related_name="role", default="Administrator")
```

Comment:
idk if let admin be the default role is the best choice, i think put the role with the less responsabilites is better

Reply:
This is only for the first sprint, will be, of course, changed at a later date.
Not even sure if we should set a default role, we should make this field required in creation.

Reply:
ok !